internal/schemautil/sensitive.go

```diff
@@ -0,0 +1,26 @@
+package schemautil
+
+import (
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+)
+
+// Those fields are not returned by API,
+// but should be in the state to make terraform work correctly
+var sensitiveFields = []string{
+	"admin_username",
+	"admin_password",
+}
+
+func copySensitiveFields(oldSrc interface{}, new []map[string]interface{}) {
```

Comment:
please add a comment

Reply:
It's just few rows upper. But added more.